v2/dtos/notification.go

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dtos
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/models"
+)
+
+// Notification and its properties are defined in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/support-notifications/2.x#/Notification
+type Notification struct {
+	common.Versionable `json:",inline"`
+	Id                 string   `json:"id,omitempty" validate:"omitempty,uuid"`
+	Created            int64    `json:"created,omitempty"`
+	Modified           int64    `json:"modified,omitempty"`
+	Category           string   `json:"category,omitempty" validate:"required_without=Labels,omitempty,oneof='SECURITY' 'SW_HEALTH' 'HW_HEALTH'"`
```

Comment:
@cloudxxx8 , @jpwhitemn do we need to rethink these Notification categories? They don't make sense to me. Why do we restrict them to these values?

Reply:
OK, let's drop the hard code category in V2